backend/instance.ts

```diff
@@ -80,6 +81,17 @@ export class Instances {
     });
   }
 
+  delete(id: number) {
+    let instance = this.instances.get(id)!;
+    this.processor.removeClient(instance.id!);
+    /*@ts-ignore*/
+    delete instance.webXdc;
```

Comment:
Deleting properties off objects is highly unlikely to be part of the real solution. I realize you're just trying stuff but it breaks fundamental expectations of the app to be deleting properties, which is why you had to add ts-ignore.